src/Nethermind/Nethermind.Runner/configs/mainnet.cfg

```diff
@@ -7,7 +7,7 @@
     "MemoryHint": 2048000000
   },
   "Network": {
-    "ActivePeersMaxCount": 100
+    "ActivePeersMaxCount": 50
```

Comment:
If this is default, remove it altogether!

Reply:
Done